oeps/oep-2.rst

```diff
@@ -0,0 +1,101 @@
+===================================
+OEP-2: Accordance to Best Practices
+===================================
+
++---------------+-------------------------------------------+
+| OEP           | :doc:`oep-2`                              |
++---------------+-------------------------------------------+
+| Title         | Accordance to Best Practices              |
++---------------+-------------------------------------------+
+| Last-Modified | 2016-04-21                                |
++---------------+-------------------------------------------+
+| Author        | Calen Pennington <cale@edx.org>           |
++---------------+-------------------------------------------+
+| Arbiter       |                                           |
++---------------+-------------------------------------------+
+| Status        | Draft                                     |
++---------------+-------------------------------------------+
+| Type          | Best Practice                             |
++---------------+-------------------------------------------+
+| Created       | 2016-04-21                                |
++---------------+-------------------------------------------+
+
+Abstract
+========
+
+Proposes a standard format for repository metadata to self-report on
+accordance to all Best Practice OEPs.
+
+Specification
+=============
+
+Each repo in the Open edX system will include a file ``.edx.yml``, with the
```

Comment:
I don't like the name.  First, why should it be a hidden file (I am torn about this myself). Second, "edx" is not the right word. At least "openedx", but maybe just "repo.yml".